src/worker/start_geneva.sh

```diff
@@ -11,8 +11,8 @@ if sudo docker ps -a --format '{{.Names}}' | grep -q "^${CONTAINER_NAME}$"; then
     exit 0
 fi
 
-GENEVA_ACCOUNT_NAME=$(jq -r '.AccountName' $GENEVA_CONFIG)
-GENEVA_MDM_ENDPOINT=$(jq -r '.MDMEndPoint' $GENEVA_CONFIG)
+GENEVA_ACCOUNT_NAME=$(jq -r '.geneva_config.AccountName' $GENEVA_CONFIG)
```

Comment:
Doesn't $GENEVA_CONFIG point to a file that no longer exists? Need to point to the moneo_config now?

Reply:
Good catch, let me address this.

Reply:
addressed